### PR TITLE
Read dim keyword from the geometry tag in the xinp file

### PR DESCRIPTION
**Pull request by VikingScientist**

### CODE REVIEW

File: Linear/main_LinEl.C [hunk 1]
```diff
@@ -160,9 +162,20 @@ int main (int argc, char** argv)
         adaptor = atoi(argv[i]+5);
     }
     else if (!infile)
+    {
       infile = argv[i];
+      if (strcasestr(infile,".xinp"))
+        if (!args.readXML(infile,false))
+          return 1;
```
Comment (akva2) [inline Linear/main_LinEl.C]: see Poisson etc and how we re-parse arguments. i know i know, but somebody might want to add more stuff and we want command line to override.

File: Linear/main_LinEl.C [hunk 2]
```diff
@@ -160,9 +162,20 @@ int main (int argc, char** argv)
...
+      if(args.dim == 1)
```
Comment (akva2) [inline Linear/main_LinEl.C]: drop these bools, use args.dim directly.

File: Linear/main_LinEl.C [hunk 3]
```diff
@@ -160,9 +162,20 @@ int main (int argc, char** argv)
...
+        oneD = true;
+      else if(args.dim == 1)
+        twoD = true;
+    }
     else
+    {
```
Comment (akva2) [inline Linear/main_LinEl.C]: heh. nope. if i cannot brace my multi-line if's you cannot brace your single-line else ;P

**Comment by VikingScientist** [comment]
jenkins build this please

**Comment by kmokstad** [comment]
Now replaced by #43.

**Pull request closed without merging**